### DISCUSSION TITLE
OTLP | Question about OTLP

### DISCUSSION THREAD

**Discussion by yonatan555** [open-telemetry/opentelemetry-cpp#1775] [Q&A]
I want to start work with OTLP in cpp 14,cmake 3.24.0, with metrics , prometheus , grpc, and without - examples, and tests.
work with static libs.
so the question is what is the options that I need to supplyto the cmake operation?

Example:
cmake -WITH_OTLP=ON , -WITH_PROMETHUES=ON WITH_GRPC_OTLP=ON. 
thanks for the help.

**Comment by owent** [comment]
You could try `cmake -DWITH_OTLP=ON  -DWITH_PROMETHUES=ON -DWITH_GRPC_OTLP=ON`.
CMake use `-D` to set variables.

**Comment by yonatan555** [comment]
ok, but I missed the grpc lib to configure, and something still missing me.
